artifacttemplates/http%3A%2F%2Fopentosca.org%2Fartifacttemplates/Ubuntu-VM_OperatingSystemInterface-w1/source/src/main/java/org/opentosca/artifacttemplates/ubuntuvm/FileHandler.java

```diff
@@ -0,0 +1,57 @@
+package org.opentosca.artifacttemplates.ubuntuvm;
```

Comment:
Doesn't it make sense to make this "reusable" for all artifact templates?

Reply:
sure but for now I dont wanna change it